tests/h/models/annotation_test.py

```diff
@@ -89,6 +96,17 @@ def test_authority_when_annotation_has_no_userid():
     assert Annotation().authority is None
 
 
+def test_default_motivations_is_empty_set(factories):
+    annotation = factories.Annotation()
+
+    assert annotation.motivations == set([])
+
+
+def test_raises_when_invalid_motivation(db_session, factories):
+    with pytest.raises(sqlalchemy.exc.ProgrammingError):
+        factories.Annotation(motivations=set(["invalid"]))
```

Comment:
To make sure the exception was raised for the reason you think it was, and not the same exception class being used for another reason, you can pass the `message` arg to `raises()`. See <https://docs.pytest.org/en/latest/assert.html?highlight=raises#assertions-about-expected-exceptions> (`message` is a regex)